src/Lists.php

```diff
@@ -0,0 +1,347 @@
+<?php
+
+namespace Klaviyo;
+
+use Klaviyo\Model\ProfileModel as profileModel;
+use phpDocumentor\Reflection\Types\Self_;
+
+/**
+ * Class Lists
+ * @package Klaviyo
+ */
+class Lists extends KlaviyoBase
+{
+    /**
+     * List endpoint constants
+     */
+    const EXCLUSIONS = 'exclusions';
+    const GROUP = 'group';
+    const LIST = 'list';
+    const LISTS = 'lists';
+    const MEMBERS = 'members';
+    const SEGMENT = 'segment';
+    const SUBSCRIBE = 'subscribe';
+
+    /**
+     * Create a new list
+     * @link https://www.klaviyo.com/docs/api/v2/lists#post-lists
+     *
+     * @param string $listName
+     * Name of list to be created.
+     *
+     * @return bool|mixed
+     */
+    public function createList( $listName )
+    {
+        $options = $this->createOptions('list_name', $listName);
+
+        return $this->v2Request( self::LISTS, $options, self::HTTP_POST );
+    }
+
+    /**
+     * Get all lists
+     * @link https://www.klaviyo.com/docs/api/v2/lists#get-lists
+     *
+     * @return bool/mixed
+     */
+    public function getLists() {
+
+        return $this->v2Request( self::LISTS );
+    }
+
+    /**
+     * Get information about a list
+     * @link https://www.klaviyo.com/docs/api/v2/lists#get-list
+     *
+     * @param $listId
+     * 6 digit unique identifier of the list
+     *
+     * @return bool|mixed
+     */
+    public function getListDetails($listId )
+    {
+        return $this->v2Request( self::LIST.'/'.$listId );
```

Comment:
we should have url path's if it's going to be a base, it should end w/ a trailing slash so we don't need to do the concatenation here